pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -86,6 +86,8 @@ var (
 	ignoreTypeMeta           = cmpopts.IgnoreFields(metav1.TypeMeta{}, "Kind", "APIVersion")
 	ignoreLastTransitionTime = cmpopts.IgnoreFields(apis.Condition{}, "LastTransitionTime.Inner.Time")
 	ignoreStartTime          = cmpopts.IgnoreFields(v1beta1.PipelineRunStatusFields{}, "StartTime")
+	ignoreCompletionTime     = cmpopts.IgnoreFields(v1beta1.PipelineRunStatusFields{}, "CompletionTime")
+	ignoreChildReferences    = cmpopts.IgnoreFields(v1beta1.PipelineRunStatusFields{}, "ChildReferences")
```

Comment:
@vsinghai this is no longer used, please remove it

Reply:
done